src/odb/src/db/dbNet.cpp

```diff
@@ -2366,6 +2371,27 @@ dbModule* dbNet::findMainParentModule() const
   return getBlock()->getTopModule();
 }
 
+bool dbNet::findRelatedModNets(std::set<dbModNet*>& modnet_set) const
```

Comment:
This doesn't return all the modnets as others may not be directly attached to a b/iterm.

Reply:
If this subset is sufficient then the api should be renamed.